<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPublishLEDInfo</class>
 <widget class="QFrame" name="CPublishLEDInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发布LED信息</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image:url(D:/WinParkUI/debug/Image/NewIcon/CommonMiddleBG-normal.jpg);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>631</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>630</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>291</height>
      </rect>
     </property>
     <column>
      <property name="text">
       <string>LED位置</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>设备CAN地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>主机IP地址</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>目标选择</string>
      </property>
     </column>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>296</x>
       <y>313</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnUnselectAll">
      <property name="geometry">
       <rect>
        <x>95</x>
        <y>3</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="text">
       <string>全取消</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSelectAll">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>3</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image:url(none);</string>
      </property>
      <property name="text">
       <string>全选择</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>235</x>
     <y>620</y>
     <width>260</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnDownload">
    <property name="geometry">
     <rect>
      <x>94</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="text">
     <string>发布信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownloadTime">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url( );</string>
    </property>
    <property name="text">
     <string>下传时间</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>5</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);
color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>49</x>
     <y>430</y>
     <width>631</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(none);</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>10</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url( );</string>
    </property>
    <property name="text">
     <string>信息内容（不超过20个字符，空格及标点必须为全角）</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="edtLedInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>35</y>
      <width>611</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image:url(none);</string>
    </property>
    <property name="plainText">
     <string>欢迎光临</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="lblClose">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>7</y>
     <width>30</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
